modules/@angular/upgrade/src/aot/upgrade_module.ts

```diff
@@ -78,8 +78,7 @@ export class UpgradeModule {
             //   }
             // ]);
           }
-      }])
-      .config(config);
+      }]);
```

Comment:
Good spot that we don't need to run the config block twice!
But I think that `IAngularBootstrapConfig` is a dodgy type and would prefer it to be `IInjectable`